ui/app/pages/permissions-connect/permissions-connect.component.js

```diff
@@ -107,6 +110,11 @@ export default class PermissionConnect extends Component {
               })
             }}
             addressLastConnectedMap={addressLastConnectedMap}
+            cancelPermissionsRequest={requestId => {
+              rejectPermissionsRequest(requestId)
```

Comment:
It looks like this would blow up if `requestId` was falsy. Maybe this should be in an if block? `if (requestId) { ... }`